working-groups/query-standardization/dsl-survey/ppl/README.md

```diff
@@ -0,0 +1,256 @@
+# PPL for CNCF Unified query language
+
+In Response to the [DSL Survey requirements](https://github.com/cncf/tag-observability/tree/main/working-groups/query-standardization/dsl-survey) the next document addresses these concerns and emphasize the advantages of PPL language that should become the best option matching the new Observability query language:
+
+### **Short Description:**
+
+PPL (Pipe Processing Language) is a query language used in OpenSearch, allowing users to interact with their data in a more expressive and readable manner. It features a structure similar to Unix's pipe commands, where operations are chained together using a pipe symbol (|). This design enables users to perform complex queries in a more intuitive way, such as filtering, transforming, and aggregating data. By leveraging PPL, developers can enhance their ability to extract insights from large datasets within the OpenSearch ecosystem.
+
+The PPL (Pipe Processing Language) architecture within OpenSearch is built to facilitate querying and data manipulation in a way that resembles Unix's pipe commands.
+
+### Overview
+
+1. **Design Goals**:
+
+    * **URI Friendliness**: Enables intuitive querying through RESTful API.
+    * **Minimal Syntax**: Human-readable and concise.
+    * **Advanced Analytical Capabilities**: Supports aggregation, filtering, etc.
+    * **Easily extensible :** support simple and build in extension mechanics
```

Comment:
Any examples of a useful extension? Elsewhere it would good to have a description of the design decisions involved in what features are in the core language vs what should be an extension and why extensions are supported.